library/src/G4CMPCrystalGroup.cc

```diff
@@ -10,6 +10,7 @@
 //
 // 20170728  Change function args "alpha, beta, gamma" to "al, bt, gm" (-Wshadow)
 // 20240426  S. Zatschler -- Add explicit fallthrough statements to switch cases
+// 20240924  I. Hernandez -- Add an if statement to include a rhombohedral Crystal group and filling more explicitly the function G4CMPCrystalGroup::FillOrthorhombic()
```

Comment:
Please limit comment lines to 80 columns or less (look at all the preceding entries).  Just break your comment into multiple lines.